.github/CODEOWNERS

```diff
@@ -276,7 +276,7 @@
 
 # PRLabel: %Digital Twins
 # ServiceLabel: %Digital Twins %Service Attention
-/sdk/digitaltwins/                              @drwill-ms @azabbasi @johngallardo
+/sdk/digitaltwins/                              @johngallardo
```

Comment:
Same as Java comment; should have > 1 owner

Reply:
```suggestion
/sdk/digitaltwins/                              @johngallardo @efriesner
```